src/data/utility/__test__/util.test.js

```diff
@@ -4,7 +4,7 @@ import {
   convertObjectIntoKeyValueArray,
   customError,
   mergeArraysWithObjects,
-  getUserDateTimefromUTC,
```

Comment:
@julianideamotive shouldn't we still also test the other function?

Reply:
@maartenhemmes tests added